.env.local

```diff
@@ -1,4 +1,7 @@
+export ERP_SERVICE=http://0.0.0.0:3000
 export SMTP_USER_NAME=XXX
 export SMTP_PASSWORD=XXX
 export SMTP_SERVER=smtp.gmail.com
-export SMTP_SERVER_PORT=587
\ No newline at end of file
+export SMTP_SERVER_PORT=587
+export SD_URL=https://servicediscovery.ng.bluemix.net
+export SD_AUTH=XXX
```

Comment:
@cphrmky is it best practice to have a `\n` at the end of non-`.py` files as well?